File: fix_i18n.py
with open(r'C:\Users\Administrator\.openclaw\workspace\portraitpay\src\lib\i18n\translations.ts','r',encoding='utf-8') as f:
    content = f.read()

old_zh = '''      deleteAccountDesc: "删除账户将清除所有数据，且无法恢复",
      namePlaceholder: "输入您的显示名称",
      // Wallet binding'''

new_zh = '''      deleteAccountDesc: "删除账户将清除所有数据，且无法恢复",
      deleteAccountBtn: "删除账户",
      namePlaceholder: "输入您的显示名称",
      // Wallet binding'''

old_en = '''      deleteAccountDesc: "Deleting your account will remove all data and cannot be recovered",
      namePlaceholder: "Enter a display name",
      // Wallet binding'''

new_en = '''      deleteAccountDesc: "Deleting your account will remove all data and cannot be recovered",
      deleteAccountBtn: "Delete Account",
      namePlaceholder: "Enter a display name",
      // Wallet binding'''

content = content.replace(old_zh, new_zh, 1)
content = content.replace(old_en, new_en, 1)

with open(r'C:\Users\Administrator\.openclaw\workspace\portraitpay\src\lib\i18n\translations.ts','w',encoding='utf-8') as f:
    f.write(content)

print('done')

File: fix_i18n.js
const fs = require('fs');
let content = fs.readFileSync('C:\\Users\\Administrator\\.openclaw\\workspace\\portraitpay\\src\\lib\\i18n\\translations.ts', 'utf8');

const oldZh = `      deleteAccountDesc: "删除账户将清除所有数据，且无法恢复",
      namePlaceholder: "输入您的显示名称",
      // Wallet binding`;

const newZh = `      deleteAccountDesc: "删除账户将清除所有数据，且无法恢复",
      deleteAccountBtn: "删除账户",
      namePlaceholder: "输入您的显示名称",
      // Wallet binding`;

const oldEn = `      deleteAccountDesc: "Deleting your account will remove all data and cannot be recovered",
      namePlaceholder: "Enter a display name",
      // Wallet binding`;

const newEn = `      deleteAccountDesc: "Deleting your account will remove all data and cannot be recovered",
      deleteAccountBtn: "Delete Account",
      namePlaceholder: "Enter a display name",
      // Wallet binding`;

content = content.replace(oldZh, newZh);
content = content.replace(oldEn, newEn);

fs.writeFileSync('C:\\Users\\Administrator\\.openclaw\\workspace\\portraitpay\\src\\lib\\i18n\\translations.ts', content, 'utf8');
console.log('done');
